CMakeLists.txt

```diff
@@ -124,7 +124,7 @@ endif()
 
 ####################################### Program Info #######################################
 
-set(Launcher_APP_BINARY_NAME "polymc" CACHE STRING "Name of the Launcher binary")
+set(Launcher_APP_BINARY_NAME "PolyMC" CACHE STRING "Name of the Launcher binary")
```

Comment:
Instead of changing the default binary name, replace all instances of `PolyMC` in `Launcher.in` with `@Launcher_APP_BINARY_NAME`. This provides more flexibility, e.g. for when a different name is desired.